src/runtime/config/configuration-clh.toml.in

```diff
@@ -56,6 +56,15 @@ disable_selinux=@DEFDISABLESELINUX@
 # (default: true)
 disable_guest_selinux=@DEFDISABLEGUESTSELINUX@
 
+# disable applying AppArmor profile on the container process
+# If set to false, the kata-default profile is applied to the container process by default.
+# The kata-default profile is based on the containerd's default profile:
+# https://github.com/containerd/containerd/blob/main/contrib/apparmor/template.go
```

Comment:
It's better to use a fixed tag(v1.7.x) or commit id instead of a branch name.

Reply:
Used the 1.7.3 tag. Thanks!